include/room_message_types.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+//
+
+#pragma once
+
+#include "MessageIdentifiers.h"
+
+enum RoomMessageTypes
```

Comment:
See the document for the packets we decided on, there's an enum RoomMessageTypes somewhere in there